ci/logstash_releases.json

```diff
@@ -5,7 +5,7 @@
     "7.x": "7.7.0"
   },
   "snapshots": {
-    "7.x": "7.8.0-SNAPSHOT",
+    "7.x": "7.9.0-SNAPSHOT",
```

Comment:
Can we also update the releases for `7.x` to `7.8.1` and `6.x` to `6.8.11`?

Reply:
Done.